gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu May 28 13:12:23 PDT 2020
+#Tue Oct 13 12:06:18 PDT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
Is the thought here that hybrid devs probably don't care about native source and documentation?

Reply:
The default `Update` action in the latest `Android Studio` has switched from `all` to `bin` as well - I noticed this while updating another project.

Reply:
Oh, good to know.